fxpubsub/probe.go

```diff
@@ -0,0 +1,43 @@
+package fxpubsub
```

Comment:
can I ask you to move the probe in a `fxpubsub/healthcheck` package ? for consistency with [other libs](https://github.com/ankorstore/yokai/tree/main/orm/healthcheck)